lib/funky/funky_object.rb

```diff
@@ -0,0 +1,16 @@
+module Funky
+  class FunkyObject
```

Comment:
@bruce-fs `Funky::FunkyObject` sounds redundant and not too informative about this class. Can you think of alternatives?